ui/nuxeo-document-comments/nuxeo-document-comment-thread.js

```diff
@@ -0,0 +1,293 @@
+              id="inputContainer"
+              placeholder="[[_computeTextLabel(level, 'writePlaceholder', null, i18n)]]"
+              value="{{text}}"
+              max-rows="[[_computeMaxRows()]]"
+              no-label-float
+              on-keydown="_checkForEnter"
+            >
+            </paper-textarea>
+            <template is="dom-if" if="[[!_isBlank(text)]]">
+              <iron-icon
+                id="submit"
+                name="submit"
+                class="main-option opaque"
+                icon="check"
+                on-tap="_submitComment"
+              ></iron-icon>
+              <nuxeo-tooltip for="submit">[[i18n('comments.submit.tooltip')]]</nuxeo-tooltip>
+              <iron-icon name="clear" class="main-option opaque" icon="clear" on-tap="_clearInput"></iron-icon>
+            </template>
+          </div>
+        </template>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-document-comment-thread';
+    }
+
+    static get properties() {
+      return {
+        uid: {
+          type: String,
+          observer: '_refresh',
+        },
+
+        comments: {
+          type: Array,
+          value() {
+            return [];
+          },
+        },
+
+        level: {
+          type: Number,
+          value: 1,
+        },
+
+        pageSize: {
+          type: Number,
+          readOnly: true,
+          value: 10,
+        },
+
+        allCommentsLoaded: {
+          type: Boolean,
+          readOnly: true,
+          value: false,
+        },
+
+        total: {
+          type: Number,
+          readOnly: true,
+          value: 0,
+        },
+      };
+    }
+
+    connectedCallback() {
+      super.connectedCallback();
+      this.addEventListener('delete-comment', this._handleDeleteEvent);
+      this.addEventListener('edit-comment', this._handleEditEvent);
+      this.addEventListener('comments-changed', this._handleCommentsChange);
+    }
+
+    disconnectedCallback() {
+      this.removeEventListener('delete-comment', this._handleDeleteEvent);
+      this.removeEventListener('edit-comment', this._handleEditEvent);
+      this.removeEventListener('comments-changed', this._handleCommentsChange);
+      super.disconnectedCallback();
+    }
+
+    focusInput() {
+      this.$$('#inputContainer').focus();
+    }
+
+    _checkForEnter(e) {
+      if (e.keyCode === 13 && e.ctrlKey) {
+        if (!this._isBlank(this.text)) {
+          this._submitComment();
+        }
+      }
+    }
+
+    _clearInput() {
+      this.text = '';
+    }
+
+    _clearRequest() {
+      this.$.commentRequest.data = {};
+      this.$.commentRequest.headers = {};
+      this.$.commentRequest.params = {};
+    }
+
+    _fetchComments(loadAll) {
+      this._clearRequest();
+      this.$.commentRequest.params = {
+        pageSize: loadAll ? 0 : this.pageSize,
+        currentPageIndex: 0,
+      };
+      this.$.commentRequest.headers = {
+        'X-NXfetch.comment': 'repliesSummary',
```

Comment:
Please use `fetch-comment` header instead (see [NXP-28075](https://jira.nuxeo.com/browse/NXP-28075) for more).